mdtraj/geometry/include/math_patch.h

```diff
@@ -1,11 +1,14 @@
 // Math functions for old versions of visual studio
-
+#ifndef __MATH_PATH_H_
```

Comment:
nit pick: This should probably be `MATH_PATCH_H`. (path --> patch) double underscores are (supposed to be) reserved for compiler vendors